server.js

```diff
@@ -0,0 +1,90 @@
+// Add Express and CORS
+import express from "express";
+import cors from "cors";
+const app = express();
+const port = 5000
+import { join, dirname } from 'path'
+import { Low, JSONFile } from 'lowdb'
+import { fileURLToPath } from 'url'
+import { v4 as uuidv4 } from 'uuid';
+// --------------------------------------------------
+
+
+// LOWDB SETUP
+const __dirname = dirname(fileURLToPath(import.meta.url));
+// Use JSON file for storage
+const file = join(__dirname, 'data/db.json')
+const adapter = new JSONFile(file)
+const db = new Low(adapter)
+// Read data from JSON file, this will set db.data content
+await db.read()
+// Set default data
+db.data ||= { users: [] }
+// --------------------------------------------------
+
+
+// Express and CORS middleware
+app.use( express.json() );
+app.use( cors() );
+
+
+// Initial array of users
+// let users = [
+//     { _id: "u1", username: "user1", password: "pw1"},
+//     { _id: "u2", username: "user2", password: "pw2"},
+//     { _id: "u3", username: "user3", password: "pw3"},
+// ];
+
+
+// Define a POST route
+app.post("/signup", async (req, res) => {
+
+    console.log(req.body);
+    
+    // Create new user on request
+    let user = {
+        _id: uuidv4(),
+        ...req.body
+    };
```

Comment:
You are so nice to allow duplicate users :) But because that was not a requirement of the exercise to prevent duplicates, I cannot criticize anything :)